deltachat-ios/Controller/AccountSetup/Instand Onboarding/InstantOnboardingViewController.swift

```diff
@@ -282,14 +283,8 @@ class InstantOnboardingViewController: UIViewController {
             guard let self else { return }
 
             let qrCodeData = self.qrCodeData ?? "dcaccount:https://nine.testrun.org/new"
-            let success = self.dcContext.setConfigFromQR(qrCode: qrCodeData)
             DispatchQueue.main.async {
-                if success {
-                    self.dcAccounts.stopIo()
-                    self.dcContext.configure()
-                } else {
-                    progressAlertHandler.updateProgressAlert(error: self.dcContext.lastErrorString, completion: nil)
```

Comment:
Have you tested that showing an error still works?

Reply:
`addTransportFromQr` does not return an error state, or? how shall i check for errors? do we get the same events as by `configure()`

Reply:
k, i will have a closer look. what is a good way to force an error, so i know what to test for? passing "foo bar" or bad accountId to addTransportFromQr?

Reply:
Enter email address alice@example.net and password 1234. Or disable Wi-Fi on your phone.